src/PKSim.Assets.Images/PKSim.Assets.Images.csproj

```diff
@@ -14,6 +14,11 @@
     <Version Condition="'$(Version)' == ''">12.1.0</Version>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Resources\SmallMoleculesStructure.png" />
+    <None Remove="Resources\TwoPoresModelStructure.png" />
+  </ItemGroup>
+
```

Comment:
why do we need this?

Reply:
That's all done automatically when you select `EmbeddedResource` for the type

Reply:
hmm, ok